### PR TITLE
chore(go): update echo handler

### PR DESCRIPTION
**Pull request by appleboy**
as title.

### CODE REVIEW

File: go/echo/go.mod [hunk 1]
```diff
@@ -1,3 +1,5 @@
 module main
 
+go 1.14
```
Comment (waghanza) [inline go/echo/go.mod:3]: could you remove that ?

Comment (appleboy) [reply go/echo/go.mod]: @waghanza Done

**Comment by appleboy** [comment]
@waghanza You mean that merge multiple changes to single PR?

**Comment by waghanza** [comment]
yep

**Pull request closed without merging**